practice-app/api/urls.py

```diff
@@ -3,6 +3,7 @@
 from . import views
 
 urlpatterns = [
-    path('helloworld/', views.HelloWorldView.as_view(), name='helloworld'),
+    path('helloworld/', views.HelloWorldView.as_view(), name='helloworld'), 
+    path('UptodateParity/', views.UptodateParityView.as_view(), name='UptodateParity'),
```

Comment:
as the url path, we planned to stick with this structure(since it's the convention and feels more convenient):
'/parity/' for listing the parities, for other parity functionalities:
'/parity/latest/' for the latest data, '/parity/history' etc. . 
Hence, you might want to change /uptodateparity for the sake of consistency

Reply:
I agree with @mertyg . By the way, URLs are case-insensitive. Having uppercase letters in url patterns does not make a difference.